deploy/kubernetes/kubectl/che-kubernetes.yaml

```diff
@@ -270,6 +270,8 @@ items:
               configMapKeyRef:
                 key: CHE_INFRA_KUBERNETES_SERVER__STRATEGY
                 name: che
+          - name: CHE_WORKSPACE_FEATURE_API
+            value: "http://feature-api-service:3000"
```

Comment:
Wouldn't it better to move a value to the config map?

Reply:
Agree, will update this